zmon_worker_monitor/builtins/plugins/http.py

```diff
@@ -148,6 +154,7 @@ def __init__(
             allow_redirects=None,
             verify=True,
             oauth2=False,
+            oauth2_token='uid',
```

Comment:
I feel the name could be confusing here, as it implies a ``token value`` rather than a ``token name`` or identifier. May be ``oauth2_token_name`` could be clearer?